src/Illuminate/Database/Concerns/ManagesTransactions.php

```diff
@@ -38,11 +36,25 @@ public function transaction(Closure $callback, $attempts = 1)
                 $this->handleTransactionException(
                     $e, $currentAttempt, $attempts
                 );
+                continue;
```

Comment:
Add a white line above.

```suggestion

                continue;
```

Reply:
Fixed in 2ee59b1.